manifests/init.pp

```diff
@@ -63,6 +63,7 @@
   $watches           = {},
   $checks            = {},
   $acls              = {},
+  $umask             = '0022',
 ) inherits consul::params {
 
```

Comment:
Yea but, whatever your input in has to be readable by all the other outputs? Your numeric re there seems very sane to me.